nano/node/transport/tcp_server.cpp

```diff
@@ -402,18 +402,18 @@ void nano::transport::tcp_server::received_message (std::unique_ptr<nano::messag
 	else
 	{
 		// Error while deserializing message
-		debug_assert (message_deserializer->status != transport::message_deserializer::parse_status::success);
+		debug_assert (message_deserializer->status != transport::parse_status::success);
 
-		node->stats.inc (nano::stat::type::error, nano::to_stat_detail (message_deserializer->status));
-		if (message_deserializer->status == transport::message_deserializer::parse_status::duplicate_publish_message)
+		node->stats.inc (nano::stat::type::error, to_stat_detail (message_deserializer->status));
+		if (message_deserializer->status == transport::parse_status::duplicate_publish_message)
 		{
-			node->stats.inc (nano::stat::type::filter, nano::stat::detail::duplicate_publish);
+			node->stats.inc (nano::stat::type::filter, nano::stat::detail::duplicate_publish_message);
 		}
 		else
 		{
 			// Avoid too much noise about `duplicate_publish_message` errors
 			node->logger.debug (nano::log::type::tcp_server, "Error deserializing message: {} ({})",
-			nano::to_string (message_deserializer->status),
```

Comment:
Why not use `nano::transport::to_string` and make it obvious which functions will be called as we do everywhere else? I don't understand why we would change style in this case.

Reply:
The fact that this line had to be changed at all is a reason that we shouldn't use `nano::transport::to_string`

Reply:
I don't disagree but when I joined the project and I queried why we use fully qualified function names everywhere I was told that's the rule and we do it for clarity. So, we don't have that rule anymore?

Reply:
I think you're trying too hard to make a point here, even when using `std::` it's a good rule to always use fully qualified namespace, but there are cases where you want to instead rely on adl, eg. `swap(..)` idiom.

Reply:
Ok